client/src/components/Pages/recherche/SearchHeader/Filter/Filter.module.scss

```diff
@@ -39,11 +45,31 @@
       margin: 0 0 0 u(2);
     }
   }
+}
 
-  &.selected {
-    background: $bleuCharte !important;
-    color: $white !important;
-  }
+.checkbox {
+  flex: 1 0 0;
+  align-items: start;
+}
+
+.countContainer {
+  display: flex;
+  padding-top: 4px;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.count {
+  color: var(--tokens-light-text-mention-grey, #666);
+  text-align: right;
+
+  /* 2.Corps de texte/XS - Texte mention/Desktop & Mobile - Regular */
```

Comment:
for this we have an utility to simplify. You can find the mixin `dsfr-text` in `src/scss/mixins/_text.scss`. 
It can be used like that: 
```scss
@include dsfr-text("very-small");
```